packages/frontend/core/src/components/affine/setting-modal/general-setting/experimental-features/config.ts

```diff
@@ -0,0 +1,72 @@
+import type { AppSetting } from '@toeverything/infra';
+
+export type BuildChannel = 'stable' | 'beta' | 'canary' | 'internal';
+
+export type FeedbackType = 'discord' | 'email' | 'github';
+
+export type Flag<K extends string> = Partial<{
+  [key in K]: {
+    displayName: string;
+    displayChannel: BuildChannel[];
+    description?: string;
+    feedbackType?: FeedbackType;
+    displayInClientOrWeb?: 'client' | 'web';
+  };
+}>;
+
+// feature flag -> display name
+export const blocksuiteFeatureFlags: Flag<keyof BlockSuiteFlags> = {
+  enable_expand_database_block: {
+    displayName: 'Enable Expand Database Block',
+    description:
```

Comment:
no plan for i18n?

Reply:
To be honest, these signs are probably only temporary and I don’t think international support is necessary.